tests/system/providers/openshift.py

```diff
@@ -0,0 +1,115 @@
+import base64
+import os
+import subprocess
+import sys
+import time
+import urllib2
+
+
+def start():
+    print "Stopping existing origin container, if any..."
+    try:
+        subprocess.check_call('docker rm -f origin', shell=True)
+    except subprocess.CalledProcessError:
+        pass
+    if not (os.path.exists('/usr/bin/kubectl') or
+            os.path.exists('/usr/local/bin/kubectl')):
+        print "No kubectl bin exists? Please install."
+        sys.exit(1)
+
+    cmd = """
+docker run -d --name "origin" \
+  --privileged --pid=host --net=host \
+  -v /:/rootfs:ro -v /var/run:/var/run:rw -v /sys:/sys -v /var/lib/docker:/var/lib/docker:rw \
+  -v /var/lib/origin/openshift.local.volumes:/var/lib/origin/openshift.local.volumes \
+  openshift/origin start"""
```

Comment:
we shouldn't use latest image. This should be latest stable version, currently  `openshift/origin:v1.2.1`